.github/workflows/python-ci.yml

```diff
@@ -18,9 +18,13 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
+      env:
+        PIP_INDEX_URL: https://pypi.pacificclimate.org/simple
       run: |
+        sudo apt-get install libudunits2-dev libnetcdf-dev
         pip install -r requirements.txt
         pip install -e .[dev]
+        Rscript install_pkgs.R
```

Comment:
Only because I just did Sangwon's PR, but doesn't `install_pkgs.R` require an `r_requirements.txt` file as a command line argument? Maybe `install_pkgs.R` is different between the two workflows?

I.e., see: https://github.com/pacificclimate/chickadee/blob/cda2e983005076c9a9bf6a61b521e2c558582e64/.github/workflows/python-ci.yml#L27